packages/language/languages/en.ts

```diff
@@ -404,7 +404,9 @@ export default {
   payPalLanguage: {
     connectPayPal: `Connect PayPal`,
     connectSuccess: `PayPal enabled successfully.`,
+    disconnectPayPal: `Disconnect PayPal`,
```

Comment:
to prevent this PR from breaking other languages, can you add `disconnectPayPal` to all of the other language files? You can keep the text in English, and it's good to have at least the field in all the files